README.md

```diff
@@ -1 +1,77 @@
-# covid-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Covid Tracker Application
+
+![screenshot](./demo.png)
+![screenshot](./demo1.png)
+![screenshot](./demo2.png)
+
+# Project Description
+
+> The Covid-stats-app allows users to have access to the latest covid data in Us. They can also filter by region name, You can also get more informations on that specific region by clicking on it.
+## Built With
+
+- Major languages
+    - JavaScript
+    - HTML & CSS
+- Frameworks
+    - React
+- State Management
+    - Redux
+- Text editor
+    - VSCode
+- Api
+    - - [Narrativa API](https://covid19tracking.narrativa.com/index_en.html)
+
+## Demo
+- [live demo] (https://deploy-preview-2--infallible-mccarthy-668a73.netlify.app/)
+
+## video demo
+https://www.loom.com/share/9069364c5149416096d9bba96608c045
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have NodeJS installed
+- Have Git Bash installed
+- Have a code editor
+
+### Setup
+
+- Get this repository on your computer using ```git@github.com:Gbengacode/covid-app.git``` on any folder
+- Open the created folder on your code editor
+
+### Install
+
+- Run ```npm install``` using your integrated console
+
+### Usage
+
+- You can launch the React app using ```npm start``` on your integrated console
+
```

Comment:
- [ ] [OPTIONAL] Since you added tests for your app I think it could be better to add the command to run the tests `npm test`